Rubberduck.Core/UI/Command/MenuItems/CommandBars/AppCommandBarBase.cs

```diff
@@ -181,9 +200,7 @@ public void RemoveCommandBar()
                     Logger.Debug("Removing commandbar.");
                     RemoveChildren();
                     Item.Delete();
-                    Item.Dispose();
                     Item = null;
-                    Parent?.Dispose();
                     Parent = null;
```

Comment:
Could we get a comment here that nulling them out also disposes them? That helps me not remove these lines